### PR TITLE
Convert /doc

### PR DESCRIPTION
**Pull request by jwrober**
Finally convert all the docs in `/doc` to `rst` files in `/docs`

### CODE REVIEW

File: docs/Developing/bugs.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+* If the first two items don't work, then feel free to submit a new issue. We have implemented GitHub issue
+  templates so they will walk you through the major aspects of a new issue. It is important to note that the
+  more details you can provide, the better. If a screen shot is available, then by all means supply it to us.
+
+  * If you are an advanced hacker, you can also providde a core dump or stack trace to better help the
+    developers of Freeciv21 understand the nature of the issue.
+
+  * If there is an issue in a translation, please report it. We are still getting organized with
+    internationalization (i18n) support for the client and server. The more we know the better.
```
Comment (lmoureaux) [inline docs/Developing/bugs.rst:28]: Mention our shiny issue templates?

Comment (jwrober) [reply docs/Developing/bugs.rst:28]: We don't have a template for i18n...

Comment (lmoureaux) [reply docs/Developing/bugs.rst:28]: This was a general comment about the whole text above, not this particular line, sorry :)

Comment (jwrober) [reply docs/Developing/bugs.rst:37]: added

**Review by lmoureaux — APPROVED** [review]
I thought this typo was fun
image

**Pull request merged**